src/graph/two_player_graph.py

```diff
@@ -365,6 +365,7 @@ def __call__(self,
         :return: A concrete/active instance of the TwoPlayerGraph
         """
         self._instance = TwoPlayerGraph(graph_name, config_yaml, save_flag)
+        self._instance._graph = nx.MultiDiGraph(name=graph_name)
```

Comment:
This is not necessary.
It's being taken care of when constructing TwoPlayerGraph

See
https://github.com/aria-systems-group/regret_synthesis_toolbox/blob/331b554d88f1ed1a1d8d51aee1ba6d6cabe3eec4/src/graph/two_player_graph.py#L24